script.js

```diff
@@ -0,0 +1,395 @@
+      187: '=',
+      81: 'й',
+      87: 'ц',
+      69: 'у',
+      82: 'к',
+      84: 'е',
+      89: 'н',
+      85: 'г',
+      73: 'ш',
+      79: 'щ',
+      80: 'з',
+      219: 'х',
+      221: 'ъ',
+      65: 'ф',
+      83: 'ы',
+      68: 'в',
+      70: 'а',
+      71: 'п',
+      72: 'р',
+      74: 'о',
+      75: 'л',
+      76: 'д',
+      186: 'ж',
+      222: 'э',
+      90: 'я',
+      88: 'ч',
+      67: 'с',
+      86: 'м',
+      66: 'и',
+      78: 'т',
+      77: 'ь',
+      188: ',',
+      190: '.',
+      191: '/',
+      226: '\\',
+    },
+
+  },
+  initInput() {
+    this.elements.input = document.createElement('textarea');
+    this.elements.input.classList.add('textarea');
+    this.elements.input.id = 'use-keyboard';
+    this.elements.input.setAttribute('autofocus', 'true');
+    document.body.appendChild(this.elements.input);
+  },
+
+  initKeyboard() {
+    // create main elements
+    this.elements.main = document.createElement('div');
+    this.elements.keysContainer = document.createElement('div');
+    // setup main elements
+    this.elements.main.classList.add('keyboard');
+    this.elements.keysContainer.classList.add('keyboard--keys');
+    this.elements.keysContainer.appendChild(this.createKeys());
+    this.elements.keys = this.elements.keysContainer.querySelectorAll('.keyboard--key');
+
+    // add to DOM
+    document.body.appendChild(this.elements.main);
+    this.elements.main.appendChild(this.elements.keysContainer);
+
+    // use keybord for element whith id use-keyboard
+    const input = document.getElementById('use-keyboard');
+    this.open((currentValue) => { input.value = currentValue; });
+  },
+
+  changeLanguageWithKeyboard() {
+    document.addEventListener('keydown', (event) => {
+      if (event.shiftKey && event.keyCode === 32) {
+        this.changeLanguage();
+      }
+    });
+  },
+
+  highlightKey() {
+    document.addEventListener('keydown', (event) => {
+      document.getElementById(event.keyCode).classList.add('keyboard--key-pushed');
+    });
+
+    document.addEventListener('keyup', (event) => {
+      document.getElementById(event.keyCode).classList.remove('keyboard--key-pushed');
+      this.properties.value = document.getElementById('use-keyboard').value;
+    });
+  },
+
+  saveSelectedLanguage() {
+    sessionStorage.setItem('languageForKeyboard', this.properties.lng);
+  },
+
+  takeLanguage() {
+    this.properties.lng = sessionStorage.getItem('languageForKeyboard') ? sessionStorage.getItem('languageForKeyboard') : 'EN';
+  },
+
+  defineChar(lng, keyEventCode) {
+    if (lng === 'EN') {
+      return this.properties.alphabetEn[keyEventCode];
+    }
+    if (lng === 'RU') {
+      return this.properties.alphabetRu[keyEventCode];
+    }
+    return this;
+  },
+
+  createKeys() {
+    const fragment = document.createDocumentFragment();
+
+    // creates HTML for icons
+    const createIconHTML = (iconName) => `<i class="material-icons">${iconName}</i>`;
```

Comment:
It would be better to use DOM and NODE API methods for creating elements as you did before.